config/samples/v2/dspa_all_fields.yaml

```diff
@@ -0,0 +1,214 @@
+# This file should not be used to deploy a DataSciencePipelinesApplication
```

Comment:
Not for this file per-se, but perhaps we should add a simple README.md to the `v2` folder to explain what each of the directories/samples are for or exibit?

Reply:
Good idea. I created a follow up issue: https://issues.redhat.com/browse/RHOAIENG-2121